tests/test_markdown.py

```diff
@@ -21,7 +21,7 @@ def test_get_file_names(tmp_path):
     para_1.write_text("text")
     para_2.write_text("text")
     output = md.get_file_names(directory)
-    expected = [f"{directory}/hello.md", f"{directory}/world.md"]
+    expected = [f"{directory}\\hello.md", f"{directory}\\world.md"]
```

Comment:
@ColemanKobe Changing to `\\` is probably not gonna work on macOS and Ubuntu. Check the `os.path.sep` we were talking about.